OWNERS_ALIASES

```diff
@@ -17,7 +17,6 @@ aliases:
   sig-auth-authenticators-reviewers:
     - deads2k
     - enj
-    - jianhuiz
```

Comment:
@liggitt - for sig-auth-authenticators-reviewers / sig-auth-encryption-at-rest-reviewers

Reply:
lgtm